### ISSUE TITLE
TextField: Borders broken in Edge with zoom out

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#1812]
### Bug Report

- __Browser__: Edge

#### Describe the issue:
If the page is zoomed out, all borders of textfield are not visible

https://dev.office.com/fabric#/components/textfield

![image](https://cloud.githubusercontent.com/assets/11937034/26198196/d7db2e1c-3be2-11e7-8b68-f56b8bdaf7d3.png)

**Comment by dzearing** [comment]
@micahgodbolt @prmittal Is this a bug in Edge, or a bug in how we're doing something? I would not think that this is something we control.

**Comment by micahgodbolt** [comment]
Yup, looks like edge mostly. There might be something we can do to make it work in edge, but filing a bug in edge seems more appropriate.  
___________________ Chrome ____________________________ | _________________________ Edge ______________________
![image](https://cloud.githubusercontent.com/assets/1434956/26225657/206b7e9e-3bdd-11e7-9533-4c8a22cd5659.png)

**Comment by prmittal** [comment]
I see that it is working fine in https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/textfield
Is there some difference in code deployed on each?

**Comment by mikewheaton** [comment]
Closing as there doesn't seem to be anything we can do here.
